.applier/group_vars/seed-hosts.yml

```diff
@@ -24,3 +28,15 @@ openshift_cluster_content:
       CLUSTER_ID: "{{ clusterid }}"
       CLOUD_REGION_ZONE: "{{ cloudregion }}c"
     namespace: openshift-machine-api
+- object: LDAP Cluster Authentication
+  content:
+    - name: LDAP Authentication
```

Comment:
Let's keep consistent indentation style in the file, looks like no indent before lists?